src/components/Books.jsx

```diff
@@ -0,0 +1,44 @@
+import AddBook from './AddBook';
+import Book from './Book';
+
+function Books() {
+  const books = [
+    {
+      id: 1,
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      progress: 64,
+      chapter: 'Chapter 17',
+    },
+    {
+      id: 2,
+      title: 'Dune',
+      author: 'Frank Herbert',
+      progress: 8,
+      chapter: 'Chapter 3: A Lesson Learned',
+    },
+    {
+      id: 3,
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+      progress: 0,
+      chapter: 'Introduction',
+    },
+  ];
+  return (
+    <section id="books">
+      {books.map((book) => (
+        <Book
+          key={book.id}
+          title={book.title}
+          author={book.author}
+          progress={book.progress}
+          chapter={book.chapter}
+        />
```

Comment:
[OPTIONAL] 
- [ ] I want to kindly suggest you create an extra component called `<Booklist>` which is imported and used inside this component placing it after the map() has been done on le 31. the Booklist component is inserted and each book is passed in as a prop to the booklist. the booklist then renders the `<book>` component and passes into it the specific book as a prop and this can be used to display the name and author of the book including the buttons 👍

Reply:
Will make changes as suggested. Thank you very much